ports/silabs/mpconfigport.mk

```diff
@@ -0,0 +1,27 @@
+LONGINT_IMPL ?= MPZ
+INTERNAL_LIBM ?= 1
+CIRCUITPY_NVM ?= 1
+CIRCUITPY_RTC ?= 1
+CIRCUITPY_WATCHDOG ?=1
+
+ifeq ($(MCU_SERIES),MG24)
+        # Audio via PWM
+        CIRCUITPY_AUDIOIO = 0
+        CIRCUITPY_AUDIOCORE ?= 0
+        CIRCUITPY_AUDIOPWMIO ?= 0
+
+        # Not yet implemented common-hal modules:
+        CIRCUITPY_AUDIOBUSIO ?= 0
+        CIRCUITPY_COUNTIO ?= 0
+        CIRCUITPY_FREQUENCYIO ?= 0
+        CIRCUITPY_I2CPERIPHERAL ?= 0
+        CIRCUITPY_ROTARYIO ?= 0
+        USB_NUM_ENDPOINT_PAIRS = 0
+        UF2_FAMILY_ID ?= 0x57755a57
```

Comment:
I'm confused; I thought these boards don't have native USB support, so there is no UF2 support?